apps/web-twig-demo/public/index.php

```diff
@@ -5,5 +5,16 @@
 require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
 
 return function (array $context) {
-    return new Kernel($context['APP_ENV'], (bool) $context['APP_DEBUG']);
+    $kernel = new Kernel($context['APP_ENV'], (bool) $context['APP_DEBUG']);
+    $kernel->boot();
+
+    // Check if HIDE_TOOLBAR parameter is set
```

Comment:
Please, document this functionality in README.

Reply:
There is no README! :O

Reply:
Anyway added.